spec/controllers/stafftools/groupings_controller_spec.rb

```diff
@@ -7,6 +7,15 @@
 
   let(:grouping) { Grouping.create(organization: organization, title: 'Grouping 1') }
 
+  let!(:group_assignment) {
```

Comment:
Did `let!` for eager-evaluation, so that the assignment is created before the delete request runs in the test. Is there a better way to do this? For some reason the test is failing, but passing on my machine. Maybe your tidied version will work